wiki/Tournaments/Official_support/en.md

```diff
@@ -167,6 +167,9 @@ Profile badge images must adhere to the following standards:
 - It must clearly display the logo, motif, or name of your tournament.
 - It must be cleanly designed and of reasonable visual quality.
 - It must not include any sort of sponsorship or promote anything besides the tournament.
+- It must not include any assets which were not specifically made for the tournament and adheres to the [Content usage guidelines](/wiki/Rules/Content_usage_permissions).
```

Comment:
"It must not include any assets which [...] adheres to the content usage guidelines"

```suggestion
- Any assets must be specifically made for the tournament and adhere to the [content usage guidelines](/wiki/Rules/Content_usage_permissions).
```

Reply:
Indifferent to actual wording, original was chosen as it is more consistent with the rest of the article points formatting-wise.

Reply:
avoiding the double-negative is def more important in this case, it can be confusing because "and adheres [...]" could be read as coming *after* the entire "not [...]" clause rather than inside of it